### ISSUE TITLE
Downloads for apps are not counted/Play Store web says app is not installed on device

### ISSUE THREAD

**Issue by artas182x** [yeriomin/YalpStore#402]
### Expected behaviour
There is missing ContentSyncRequestProto which is responsible for sending currently installed apps on device. Without this downloads are not counted for developers.

### Actual behaviour
When you open Play Store page for example on PC, it says that app is not installed or device. App downloads are not counted on Play Developer Console, so It is harmful for developers.

**Comment by LuccoJ** [comment]
It may be "harmful for developers" (though I always want to be like "citation needed" at this general idea), but if I wanted my device to phone home to Google with things like the complete list of apps I have installed... I'd just use the actual Google services and apps, not F-Droid, microG and Yalp.

**Comment by yeriomin** [comment]
That's true. Yalp Store probably is harmful for developers (just like every other web site or app which offer apks from Play Store for download). I also agree with @LuccoJ. There seems to be a conflict of interest between users and developers here.

So in my opinion this feature request...
* if implemented, brings more harm than good.
* is hardly doable. I don't know how to fill some fields of `ContentSyncRequestProto`, I don't even know the api endpoint this message is sent to. It exists in microg source, but it seems to be unused. It is not a part of purchase/download process in the Play Store app, judging by the live requests. It is probably one of those requests which Play Store app sends from time to time, regardless of what the user does.
* is not testable by me. I don't have access to Play Developer Console and don't have any apps published in Play Store.
* will not work properly with built-in account, even if everything mentioned above is solved. Obviously, multiple downloads using Yalp Store built-in account will not result in increase of the installation count. Apps are linked to account, not device.

I'd like to know how to do it (just because I'm curious) but I probably wouldn't implement it in Yalp Store. So why do you think `ContentSyncRequestProto` specifically is required for the installation to be counted? What else do you know about it?

**Comment by LuccoJ** [comment]
Since I may have sounded too harsh, anyway, I would like to point out (probably not for those already here, but for the record) that, Yalp Store downloads are still counted as *downloads* as far as Play is concerned, since Play API are used, so developers and Google may not be able to know whether I personally decide to uninstall their app, but they will still know how many people have downloaded it.

They will also get their money (and their "bought" statistics) if I decide to buy it, since Yalp Store does download paid apps, as long as they have been paid in advance using Play's interface.

To me, this seems like a reasonable compromise between developer and user interests: on the other hand, developers get their money for their apps, and Google keeps count of the downloads; users use these apps without their device being intruded into with "uninstall statistics".
